### PR TITLE
readme: update README.md

### PR DESCRIPTION
**Pull request by iK4tsu**
Signed-off-by: João Lourenço <jlourenco5691@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
...
+
+# [EnTT](https://github.com/skypjack/entt/tree/2e891fdd7b442f1a56b1e821eda99b0e8a1ddbaf)
+`EnTT` is a header-only, tiny and easy to use library for game programming and
+much more written in modern C++, mainly known for its innovative
+entity-component-system (ECS) model.
+
+![EnTT: Gaming meets modern C++](https://user-images.githubusercontent.com/1812216/42513718-ee6e98d0-8457-11e8-9baf-8d83f61a3097.png)
+
+Check or follow this amazing project at
+https://user-images.githubusercontent.com/1812216/42513718-ee6e98d0-8457-11e8-9baf-8d83f61a3097.png
```
Comment (rsubtil) [inline README.md]: You've linked to an image, probably not what you intended :stuck_out_tongue:

**Review by rsubtil — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (iK4tsu) [reply README.md]: Nice catch! Done

**Review by rsubtil — COMMENTED** [review]
Looks good to me. I just wanna make sure you're properly linking to the project, as you're linking to a specific commit.

**Comment by iK4tsu** [comment]
Yup, you were right, I was not redirecting to the correct project's page. It's fixed now. I'm going to merge this.

**Pull request merged**